README.adoc

```diff
@@ -100,7 +100,7 @@ Refinements can be activated (with `using`) at top-level (per file), inside a cl
 
 == Compatibility mode
 
-Refinements are still a quite young feature, so there’s a good chance that your gem or application will have to work on a Ruby platform that doesn’t fully support refinements yet.
+Refinements are still a young feature, so there’s a possibility that your gem or application will have to work on a Ruby platform that doesn’t fully support refinements yet.
```

Comment:
_Good chance_ sounds strange when it's actually negative.
Removed _quite_ for clarity.